lib/neo4j/core/cypher_session/adaptors/http.rb

```diff
@@ -16,7 +17,8 @@ def initialize(url, options = {})
           end
 
           def connect
-            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request))
+            @requestor = Requestor.new(@url, USER_AGENT_STRING, self.class.method(:instrument_request),
+                                       @options[:faraday_options] || @options['faraday_options'] || {})
```

Comment:
Not unless I stringify_keys! since coders will likely use :faraday_options but when config'ed via neo4j.yml (ie in Rails), the keys are strings.  There is [an explicit test for this](https://github.com/brucek/neo4j-core/blob/http_driver_config/spec/neo4j/core/cypher_session/adaptors/http_spec.rb#L34).